sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ServiceBusSessionProcessor.cs

```diff
@@ -30,10 +30,9 @@ public class ServiceBusSessionProcessor : IAsyncDisposable
         public virtual string EntityPath => InnerProcessor.EntityPath;
 
         /// <summary>
-        /// Gets the ID to identify this processor. This can be used to correlate logs and exceptions.
+        /// A unique name used to identify the processor.  If <c>null</c> or empty, a random unique value will be will be used.
```

Comment:
nit:  We should consider updating this block to remove the implied guarantee about uniqueness, since we don't have control over the value in all cases.